src/components/MissingDataFeedback/Message.tsx

```diff
@@ -0,0 +1,24 @@
+import { Alert, Box } from '@chakra-ui/react'
```

Comment:
Wait for a core maintainer stamp on this comment but:

I feel like this component has nothing related to `MissingData`

It's basically a wrapper over an informational alert, it could be more global and reusable without anything related to missing data

Reply:
agreed with @NeOMakinG again - rename this component to something more appropriate/generic, and decouple it from `assets.assetDetails.assetHeader.` specific translations